client/components/RecentsTransfer.tsx

```
@@ -36,6 +48,19 @@ const RecentTransfer = ({ data, onChange }: Data) => {
         </View>
         <Feather name="search" />
       </View>
+      <View>
+        {activePage === "recent" || 'favorite' ? (
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)